operations/meetings.md

```diff
@@ -15,9 +15,19 @@ As a distributed team, we pay a high coordination penalty for getting multiple p
 - **Take notes and make them accessible**.
   Meetings must have written notes that are accessible to others on the team.
   These must be made available to the team reasonably soon after the meeting.
-- **Meeting agendas must be available at least one working day ahead**.
-  Agendas should provide an idea for the topics of discussion and any major questions to resolve.
-  If a meeting does not have an agenda at least one working day in advance, then team members may assume it is cancelled.
+
+- **Meeting agendas must be available at least 36 hours (not counting weekends) ahead**.
+  Agendas should:
+  1. List any specific **decisions** people want to make progress on during the meeting.
+  2. Provide ideas for topics of possible discussion.
+
+  Agendas should be concrete enough that people who can not be synchronously
+  present can make comments on it, as well as make choices on wether they
```

Comment:
```suggestion
  present can make comments on it, as well as make choices on whether they
```

Reply:
wether it is whether or wether, but maybe it's weather? hmm, perhaps wether is not actually a word :D

Reply:
My friend thought a "sweater" was actually "sweather" for the longest time and it was too funny to correct

Reply:
sweeter sweather weather